src/components/main/codeView/hooks/useEditor.tsx

```diff
@@ -286,6 +286,7 @@ export default function useEditor() {
       }
     } else {
       // non-parse file save
+
       const _file = JSON.parse(JSON.stringify(ffTree[file.uid])) as TNode;

```

Comment:
Why not use structured clone here?

Reply:
I can change it in next fixes, it not fixed now because I not working on improving of this file

Reply:
sure